darts/utils/multioutput.py

```diff
@@ -0,0 +1,93 @@
+from joblib import Parallel
+from sklearn.base import is_classifier
+from sklearn.multioutput import MultiOutputRegressor as sk_MultiOutputRegressor
+from sklearn.multioutput import _fit_estimator
+from sklearn.utils.fixes import delayed
+from sklearn.utils.multiclass import check_classification_targets
+from sklearn.utils.validation import _check_fit_params, has_fit_parameter
+
+
+class MultiOutputRegressor(sk_MultiOutputRegressor):
```

Comment:
Looks good! I wonder if that could be of any use to propose this as a PR to modify the `_MultiOutputEstimator` class in sklearn directly. WDYT?

Reply:
That sounds like a good idea, I'll see what I can do 👍